src/components/PharmacyCard/PharmacyCard.tsx

```diff
@@ -1,9 +1,33 @@
 import React, { useState } from "react";
-import { Card, Banner, TextContainer, Stack } from "@shopify/polaris";
+import { format } from "date-fns-tz";
+import {
+  Card,
+  Banner,
+  TextContainer,
+  Stack,
+  DataTable,
+  Button,
+} from "@shopify/polaris";
 import { DomainsMajor, LocationMajor } from "@shopify/polaris-icons";
 import Iframe from "react-iframe";
+import {
+  VaccineAvailabilityTimeslotRequirementExpandedResponse,
+  VaccineAvailabilityRequirementsResponse,
+} from "../../apiClient";
 import { useTranslation } from "react-i18next";
 
+interface RequirementsWithAvailabilityInterface
+  extends VaccineAvailabilityRequirementsResponse {
+  numberAvailable: number;
+}
+
+interface VaccineAvailabilitiesByDateAndRequirementsInterface {
+  [key: string]: {
```

Comment:
Minor, but naming this member `date` instead of `key` would better express intent.